content/docs/start/data-pipelines.md

```diff
@@ -64,11 +64,11 @@ Please also add or commit the source code directory with Git at this point.
 </details>
 
 ```dvc
-$ dvc run -n prepare \
-          -p prepare.seed,prepare.split \
-          -d src/prepare.py -d data/data.xml \
-          -o data/prepared \
-          python src/prepare.py data/data.xml
+$ dvc stage add -n prepare \
+                -p prepare.seed,prepare.split \
+                -d src/prepare.py -d data/data.xml \
+                -o data/prepared \
+                python src/prepare.py data/data.xml
 ```
 
 A `dvc.yaml` file is generated. It includes information about the command we ran
```

Comment:
> about the command we ran

Nothing ran though.

Reply:
> We would need to run dvc exp run for that.

Why not `repro` here @shcheklein ? Experiments complicates things under-the-hood and we don't want to go into those details in this page.

Reply:
Fixed that. 

`repro` is done after more commands added. I also don't think we should mention `exp run` here.